L1Trigger/Phase2L1ParticleFlow/plugins/L1TPFProducer.cc

```diff
@@ -33,6 +34,10 @@ class L1TPFProducer : public edm::stream::EDProducer<> {
         l1tpf_impl::PFAlgoBase::VertexAlgo vtxAlgo_;  
 
         edm::EDGetTokenT<l1t::MuonBxCollection> muCands_;
+        edm::EDGetTokenT<l1t::VertexCollection> l1Vertices_;
+        //edm::EDGetTokenT<l1t::Vertex> l1Vertex_;
+
+  //const edm::EDGetTokenT<std::vector<l1t::Vertex>> l1VerticesToken_
```

Comment:
when you finish testing the PR, please also clean up here and in other places the old commented code